app/src/main/res/values/themes.xml

```diff
@@ -8,6 +8,12 @@
         <!-- Assign dialog style here. -->
         <item name="alertDialogTheme">@style/MyAlertDialogStyle</item>
         <item name="alertDialogStyle">@style/MyAlertDialogStyle</item>
+
+        <!--<item name="appRatingDialogTitleStyle">@style/MyDialogTitleStyle</item>-->
```

Comment:
Should this be commented?

Reply:
please refer to conversation with Ramzi above